include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -81,7 +81,12 @@ namespace LinearAlgebra
     class BlockVector : public BlockVectorBase<Vector<Number> >,
       public VectorSpaceVector<Number>
     {
+      /**
+       * The chunks size to split communication in update_ghost_values() and compress() calls.
```

Comment:
Can you wrap the comment line at 80 characters? Please also say that this value has been found viable in experimentation. (I think it is a very good choice.)